commons/src/main/kotlin/com/simplemobiletools/commons/extensions/Activity.kt

```diff
@@ -975,7 +983,7 @@ private fun BaseSimpleActivity.renameCasually(
     val tempFile = try {
         createTempFile(oldFile) ?: return
     } catch (exception: Exception) {
-        if (isRPlus() && exception is FileSystemException) {
+        if (isRPlus() && exception is java.nio.file.FileSystemException) {
```

Comment:
This is what was causing the renaming issue. 
We need to use the full path here (or import alias). This is because we have the star import for `java.util.*`, so Kotlin auto imports its `kotlin.io.FileSystemException` here and it conflicts with the `java.nio.file.FileSystemException` that is expected

Reply:
ah okay